### ISSUE TITLE
Test button doesn't work in Application Config

### ISSUE THREAD

**Issue by brombomb** [linuxserver/Heimdall#782]
Clicking the test button I get a JS Error:

```
app.js?v=5:721 Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at HTMLButtonElement.<anonymous> (app.js?v=5:721:22)
    at HTMLDivElement.dispatch (jquery-3.6.0.min.js:2:43064)
    at HTMLDivElement.v.handle (jquery-3.6.0.min.js:2:41048)
```
![image](https://user-images.githubusercontent.com/1136944/158862822-783e84f0-1add-48f4-b8a2-9ca20f8c6200.png)

Config currently looks like this:
![image](https://user-images.githubusercontent.com/1136944/158863022-4d4b9deb-2675-4572-990c-25608c59ecf8.png)

Doesn't matter if the config/api is working or not same issue.

Chrome 99 - Mac OS 12.2.1 "Monterey"
Heimdall version 2.4.2

**Comment by KodeStar** [comment]
Identified the issue, will be fixed in the next release.

**Comment by KodeStar** [comment]
Fixed in 2.4.3
